test/performance/shmem_perf_suite/bw_common.h

```diff
@@ -202,12 +204,18 @@ void calc_and_print_results(double end_t, double start_t, int len,
            (end_time_max - start_time_min) > 0) {
 
             total_t_max = (end_time_max - start_time_min);
+            int PEs_on_other_side = 0;
+            if (start_pe == 0) {
+                PEs_on_other_side = metric_info.sztarget;
+            } else {
+                PEs_on_other_side = metric_info.szinitiator;
+            }
 #ifdef ENABLE_OPENMP
-            bw = ((double) len * (double) multiplier * (double) metric_info.midpt / 
+            bw = ((double) len * (double) multiplier * (double) PEs_on_other_side / 
                  1.0e6 * metric_info.window_size * metric_info.trials * 
                  (double) metric_info.nthreads) / (total_t_max / 1.0e6);
 #else
-            bw = ((double) len * (double) multiplier * (double) metric_info.midpt / 
+            bw = ((double) len * (double) multiplier * (double) PEs_on_other_side / 
```

Comment:
I believe the case this calculation is missing is when two initiators are sending to the same target? We essentially want to capture the number of messages sent converted to bytes. Something along the lines of ~ each PE's number of partners * iterations * len per message. Whats the reason for calculating it differently than the bw calculation you used above it?

Reply:
Thanks Kayla. I had some doubt here as well. But, I kept it like this because of the following reason. If we use only `midpt` as before then choosing a different `l/r` would result to a wrong calculation. For example, if `midpt = 8` and `l = 5`, then using `midpt` would multiply with 8 and thus the BW calculated would be more than the correct one. 
Now, regarding the case when two initiators are sending to the same target, from initiator's perspective, each has one target and they send the `len` bytes from start to end time. So, the initiator bandwidth is multiplied by 1. For target, it has to receive the `len` bytes from two initiators within its `(end-start)` time, So, for target, the BW is calculated by multiplying by 2. I had the concern whether this is the correct way of thinking. Please let me know if it's not.

Reply:
In the above case, should the initiator bandwidth also be multiplied by 2 to get the aggregated bandwidth of 2 initiators?